DQMOffline/Alignment/src/DiMuonMassBiasMonitor.cc

```diff
@@ -33,7 +34,10 @@ DiMuonMassBiasMonitor::DiMuonMassBiasMonitor(const edm::ParameterSet& iConfig)
       MEFolderName_(iConfig.getParameter<std::string>("FolderName")),
       decayMotherName_(iConfig.getParameter<std::string>("decayMotherName")),
       distanceScaleFactor_(iConfig.getParameter<double>("distanceScaleFactor")),
-      DiMuMassConfiguration_(iConfig.getParameter<edm::ParameterSet>("DiMuMassConfig")) {}
+      DiMuMassConfiguration_(iConfig.getParameter<edm::ParameterSet>("DiMuMassConfig")) {
+  produces<DQMToken, edm::Transition::EndRun>("DQMGenerationDiMuonMassBiasMonitorRun");
+  produces<DQMToken, edm::Transition::EndLuminosityBlock>("DQMGenerationDiMuonMassBiasMonitorLumi");
```

Comment:
@mmusich Did you have a specific reason to not use the existing `DQMToken` functionality in the DQM base classes?
https://github.com/cms-sw/cmssw/blob/f3a190193507fa09d12f82a397a350766f523629/DQMServices/Core/interface/DQMEDAnalyzer.h#L57-L58
https://github.com/cms-sw/cmssw/blob/f3a190193507fa09d12f82a397a350766f523629/DQMServices/Core/interface/DQMEDHarvester.h#L79-L95

Reply:
@makortel 

> Did you have a specific reason to not use the existing DQMToken functionality in the DQM base classes?

those were a relic from a previous stage of development in which this module was and `edm::EDAnalyzer` and not a `DQMEDHarvester`. That line of development was abandoned as the legacy modules are anyway deprecated.
I removed the useless statements in https://github.com/cms-sw/cmssw/pull/39217/commits/a5bab3b6c16061202c6fb8bbb6af0c0848d8a40a as part of  #39217